homeassistant/components/esphome/entry_data.py

```diff
@@ -153,6 +162,38 @@ def _unsub() -> None:
 
         return _unsub
 
+    @callback
+    def async_register_key_static_info_remove_coro(
```

Comment:
It's a bit weird to call this `coro`. It's really just an async callback right

Reply:
Adjusted in [`3ea7ecd` (#95202)](https://github.com/home-assistant/core/pull/95202/commits/3ea7ecd6da9de1fd20f180cf15d57ac2ef4d8dec)